oioswift/common/middleware/container_hierarchy.py

```diff
@@ -179,11 +220,22 @@ def _create_key(self, req, account, container, mode, path):
         # TODO: should we increase number of objects ?
         # but we should manage in this case all error case
         # to avoid false counter
-        res = self.conn.set(key, "1")
-        if not res:
-            LOG.warn("%s: failed to create key %s", self.SWIFT_SOURCE, key)
+        if self.redis_keys_format == REDIS_KEYS_FORMAT_V1:
+            res = self.conn.set(key, "1")
+            if not res:
```

Comment:
the `if not res` could be factorized with l.229

Reply:
The log are slightly different.